### DISCUSSION TITLE
PCL的图标从哪来的

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#963] [提问] [answered]
![屏幕截图 2023-01-18 214113](https://user-images.githubusercontent.com/119654171/213186570-d0079e6b-9ac8-4910-b320-bd600e9b23a2.png)

**Comment by WForst-Breeze** [comment]
（感谢下方补充）
除了Mojang和微软徽标，其它的均是图标库

**Reply by Pigeon0v0** [reply]
是图标库

**Comment by WorldHim** [comment]
好像都能在 [iconduck.com](https://iconduck.com/) 上找到

**Comment by LTCatt** [comment, marked as answer]
https://www.iconfont.cn/
